libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.EndTurn.schema

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "End Turn",
-    "description": "End the current turn without ending the dialog.",
+    "title": "End turn",
+    "description": "End the current action without ending the dialog.",
```

Comment:
>action [](start = 36, length = 6)

this doesn't make any sense.  What was there before was highly accurate.  Why changes this?

Reply:
As below, we want to prevent repeating the word we're trying to explain in the description sentence.